implementations/rust/ockam/ockam_command/src/lib.rs

```diff
@@ -244,7 +244,7 @@ pub struct CommandGlobalOpts {
 impl CommandGlobalOpts {
     pub fn new(global_args: GlobalArgs) -> Self {
         let terminal = Terminal::from(&global_args);
-        let state = match CliState::with_default_dir() {
+        let state = match CliState::with_default_dir("test".to_string() /* FIXME */) {
```

Comment:
Short answer: yes
Long answer:
The problem is that CliState struct is designed (and used) as something that is not node-dependent, which has been fine so far, since it really manages the database which we have one per machine. But there are multiple reasons to isolate data between nodes. In this PR I isolate attributes a node has received form other nodes (inside credentials) using a dedicated field in the corresponding table, but I think:
1. There are probably other cases like that, so those tables can now easily use the same approach, since `node_name` value is already available to them.
2. I think there is also room for performance improvements - each node can have dedicated db file(s).
That being said, I see benefits in `CliState` being node-dependent. However, that transition wasn't that easy to do, since node name is not available at the stage where we instantiate `CliState`, ideally we should improve that, but for now this approach should work